### ISSUE TITLE
"Install Star Citizen" fails using Flatpak Lutris

### ISSUE THREAD

**Issue by x3rAx** [starcitizen-lug/lug-helper#56]
Hi there,

I'm trying to install Star Citizen on the Flatpak version of Lutris. After I select "Install Star Citizen" from the LUG helper, the installation wizard opens and when I click "Install", the `RSI-Setup-1.6.5.exe` is downloaded. Then wine prefix is set up and after that, the installer first says "Instlling dlls", then changes to "Command exited with code 256".

=> [Installer logs](https://paste.x3ro.net/#XQAAAQCiCgAAAAAAAAApnQgnVi7SsihgLRJzqC9ZTsHRqMIn/09MQRO09BKiH8GNUHYeDuAKmEdfdPEJpP7cW57K2b286dHPlxx8bAqrSO/N2MXhxq0ItJTcIBXm8vQY+ULxlATMrLE1FliXA/efWCbBlGf4fP4z398zkHgNH3nytb8Lyrg0gIV2DXSrZ57Ls9Y1JDtTwz9jlarZVbDbjNx5AegnzVSuGdDYelsYHP6a76oTHYUdnfsMjWl5xpxb+WoelasqpoeK40MvlS9azka5/kVK8TfJDENhk9KVDOsYbkUpb5YJe4nU0VoHaYROMHHB0+O4PrEXk4eq2OqIsvyU6MNmxYhAzYXsmAJ1YO/uD9ys1SVSsvS5GBlDN0fdIQzR6O1KcywTtro16uJjg36ZePp77bMovEqXpTbbFr5B+49dmGE2QoYGOIKD92YKPH9a/Nc55hzMiwp24utVBwQHNOLb4pMtLo0JrELN6xkenZWkJdAI8Q0rcbOmLB3N/5LpUiMaEcarVlohiLBjaNs0oUeVf+jTBmc66f0Drbk1pHqHvqZqvRkDDlkToJ7KYlkKtIIY4Xp9oRWbHFw9JjK9W0gxJ25bdxESA7quqd1Ec2RE/8e/vCvHJJAOhD4/za1krYmXclQX00/YNP/gBjfC8P3n10j01Hm8lWeONDWkpMHJkvo2vRQti6eREWiITr0/Ru/FevlgrDRh6DYZmOTaycxzZS2n0klNuTNGH2DT4YF++2oFIrV0wth0yTHSSWfXx6JTUyFRQ6ncBYVf8uWy3Tunp9c+4Wxp/r/tYNrqph7NMgM1do/b4WpiXtR9uDs7L9PXKvk4nTP3Ldy/QYyirQ1hxMtzVYq3TRddlMnxL4r88zsrAs3tIK0LyyipSiYiuvtAX1Fz/eQsHnPWiioynAOdLZrQ/SdeD1LGt5keiGxj32Gynj6KZ1KvhE68rF0CMZ1cmrPJYiGCss2mXPTDDsujoKI3N+rS+3hCHAL+JzxQsZ/LKST56nBnOwt0psye8AL5JOhtU+Lyr//zdLf9)

The issue seems to be around `wget`:

```
wget: symbol lookup error: /usr/lib/x86_64-linux-gnu/libassuan.so.0: undefined symbol: gpgrt_malloc, version GPG_ERROR_1.0
```

but I can't find any solution to this problem...

Can anyone please provide assistance?

**Comment by the-sane** [comment]
I'd recommend asking for help in our discord tech support channel; this is for reporting problems with the Helper itself. That channel is bridged through our matrix space if you don't want to use discord and there are a lot more people there who can help troubleshoot the issue.

Links are on our wiki:
https://github.com/starcitizen-lug/knowledge-base/wiki

I'm going to close this issue as that error is likely a configuration problem on your system and not an issue with the Helper.

**Comment by x3rAx** [comment]
Well.. After I wrote this issue I gave myself one more hour to find a solution before I really submitted it... Of course, less than 10 minutes later I found the solution...

What I needed to do (and what I totally forgot to try out) was to re-enable "Prefer system libraries".

From what I know about Flatpak, I guess that it always needs to be enabled when using Flatpak Lutris? If so, I think it should be added to the install instructions.
